spec/classes/server_spec.rb

```diff
@@ -328,6 +328,29 @@
         end
       end
 
+      describe 'with tls' do
+        context 'enabled' do
+          let :params do
+            {
+              tls: true,
+              tls_mode: 'requireTLS'
+            }
+          end
+
+          it { is_expected.to contain_file(config_file).with_content(%r{^net\.tls\.mode: requireTLS$}) }
+        end
```

Comment:
Can you validate the rest of the new parameters here?

Reply:
Hi @root-expert ,
I've written the tests in a new commit.
If you agree with them, I will fixup them with the commit where the parameters are introduced.